website/moneybirdsynchronization/services.py

```diff
@@ -0,0 +1,67 @@
+from payments.models import Payment
+import datetime
+
+def get_financial_account_id(api, name):
```

Comment:
what is `name` exactly?

Reply:
identifier of the financial account. Changed the name accordingly